hoot-core/src/main/cpp/hoot/core/conflate/DuplicateWayRemover.cpp

```diff
@@ -163,14 +179,34 @@ bool DuplicateWayRemover::_isCandidateWay(const ConstWayPtr& w) const
   return result;
 }
 
-void DuplicateWayRemover::_removeDuplicateNodes(shared_ptr<Way> w1, shared_ptr<Way> w2)
+void DuplicateWayRemover::_updateWayNameTags(shared_ptr<Way> way1, shared_ptr<Way> way2)
 {
+  //merge name tags for these two ways, if they aren't already identical
+  double namesScore = 0.0;
+  double weight = 0.0;
+  TagComparator::getInstance().compareNames(way1->getTags(), way2->getTags(), namesScore, weight);
```

Comment:
So, will compareNames can return a score of 1.0 even if the name tags between the two ways aren't identical?  I think given the previous comment reply I made for the line 182 comment that the score check isn't needed at all in this location anyway.  I think if the logic gets to this point the full tag merge should happen regardless....still need to try it out.

Related question:  Are the tag score checks at line 143 and 144 valid?  I'm attempting to do a check for all non-name tags being identical between the two ways....if they're not (or _strictTagMatching=false), then proceed with the merging.